qiskit/execute.py

```diff
@@ -48,6 +48,7 @@ def execute(experiments, backend,
             meas_return=MeasReturnType.AVERAGE,
             memory_slots=None, memory_slot_size=100, rep_time=None, parameter_binds=None,
             schedule_circuit=False, inst_map=None, meas_map=None, scheduling_method=None,
+            init_qubits=True,
```

Comment:
What should this default be?

Reply:
This should be `None` by default. If not set the backend will assume this to be False if resets are present and `True` otherwise if it has initialization available. Does this make sense?